src/ecommerce.ts

```diff
@@ -40,10 +42,13 @@ const mapEcommerceData = (event: MCEvent) => {
   properties.currency = data.currency
 
   properties.contents = getContents(data)
-
+  if (properties.contents?.content_id) {
+    properties.content_type = 'product'
+  }
   if (event.name === 'Products Searched') {
     properties.query = data.query
   }
+  properties.order_id = payload.order_id
```

Comment:
are we sure this order it doesn't come under ecommerce? I assume we are but I just wanted to double check

Reply:
Yes, it's ok! Thanks.